packages/date-picker/docs/Calendar.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react"
+import { Story } from "@storybook/react"
+import { usePopper } from "react-popper"
+import { enAU } from "date-fns/locale"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { DayOfWeek } from "../src/DatePicker"
+import { Calendar, CalendarProps } from "../src/DatePicker/components/Calendar"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+
+const SUPPORTED_LOCALES = ["en-US", "en-AU"]
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.datePicker}/Primitives/Calendar`,
+  component: Calendar,
+  parameters: {
+    docs: {
+      description: {
+        component: 'import { Calendar, DateInput } from "@kaizen/date-picker"',
+      },
+    },
+  },
+  argTypes: {
+    locale: {
+      options: SUPPORTED_LOCALES,
+      control: { type: "radio" },
+    },
+  },
+}
+
+const CalendarExample = (props: Partial<CalendarProps>): JSX.Element => {
+  const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(
```

Comment:
Not to be fixed here, but this shows that the Calendar component shouldn't know about popper (it should be the parent's responsibility).